config/database.yml

```diff
@@ -20,6 +20,7 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  host: <%= ENV['DATABASE_HOST'] || localhost %>
```

Comment:
It would surprise me if `localhost` is a defined variable here.

Reply:
The test runner picked this up as well. Fixed now